animation_nodes/ui/node_menu.py

```diff
@@ -390,6 +390,8 @@ def draw(self, context):
         insertNode(layout, "an_GetVertexColorLayerNode", "Get Vertex Color Layer")
         insertNode(layout, "an_InsertVertexColorLayerNode", "Insert Vertex Color Layer")
         insertNode(layout, "an_SetVertexColorNode", "Set Vertex Color")
+        insertNode(layout, "an_SetBevelVertexWeight", "Set Bevel Vertex Weight")
+        insertNode(layout, "an_SetBevelEdgeWeight", "Set Bevel Edge Weight")
```

Comment:
Perhaps we should combine all of the data setters and getters into their own sub-menu.

Reply:
Okay.

Reply:
Okay.